### PR TITLE
converted `ComputedProperty` to es6 class

### PR DESCRIPTION
**Pull request by bekzod**

### CODE REVIEW

File: packages/ember-metal/lib/alias.js [hunk 1]
```diff
@@ -22,7 +22,6 @@ export default function alias(altKey) {
 export class AliasedProperty extends Descriptor {
   constructor(altKey) {
     super();
-    this.isDescriptor = true;
```
Comment (bekzod) [inline packages/ember-metal/lib/alias.js]: `isDescriptor` is inherited now

File: packages/ember-metal/lib/mixin.js [hunk 2]
```diff
@@ -677,11 +677,10 @@ export function required() {
 }
 
 function Alias(methodName) {
-  this.isDescriptor = true;
```
Comment (bekzod) [inline packages/ember-metal/lib/mixin.js]: `isDescriptor` is inherited now

**Review by krisselden — COMMENTED** [review]
This needs to be measured

**Comment by bekzod** [comment]
benchmark results: https://cl.ly/0R1T1k0U2y15/results.pdf

**Comment by bekzod** [comment]
benchmark is wrong will update it

**Comment by bekzod** [comment]
benchmark results: https://cl.ly/1n2e0m0H030F/results.pdf

**Comment by Serabe** [comment]
restarted sauce labs

**Comment by bekzod** [comment]
isn't this good to merge :P ?

**Comment by locks** [comment]
@krisselden can you confirm the benchmark please?

**Comment by bekzod** [comment]
@krisselden maybe this is good to merge :)

**Comment by bekzod** [comment]
@rwjblue maybe this is good to merge :)

**Comment by mmun** [comment]
We need to clean up the history before merging.

**Comment by bekzod** [comment]
@mmun is this enough :P

**Pull request closed without merging**